testfiles/CmplxGlz_Daylighting_SouthVB45deg.idf

```diff
@@ -376,7 +376,7 @@
 
   WindowMaterial:ComplexShade,
     Shade_14_Layer,          !- Name
-    Venetian,                !- Layer Type
+    VenetianHorizontal,      !- Layer Type
```

Comment:
So this needs transition rules.  Document this change in the `src/Transition/InputRulesFiles/Whatever9.0Filename.md` and I will write up the Fortran transition code accordingly.